api/serializers/personal_info.py

```diff
@@ -45,3 +45,17 @@ class Meta:
         model = PersonalInfo
         fields = ("registration_telegram_bot_chat_id",)
         extra_kwargs = {"registration_telegram_bot_chat_id": {"required": True}}
+
+
+class PublicPersonalInfoSerializer(serializers.ModelSerializer[PersonalInfo]):
+    """A serializer used to return public information about a user in Tooljet views."""
```

Comment:
```suggestion
    """A serializer used to return public information about a person in Tooljet views."""
```
user is a dashboard user